src/live_restore/live_restore_fs.c

```diff
@@ -594,6 +594,20 @@ __live_restore_can_service_read(WTI_LIVE_RESTORE_FILE_HANDLE *lr_fh, WT_SESSION_
     return (FULL);
 }
 
+/*
+ * __live_restore_fh_write_destination --
+ *     Write to the destination file handle.
+ */
+static int
+__live_restore_fh_write_destination(WT_SESSION_IMPL *session, WTI_LIVE_RESTORE_FILE_HANDLE *lr_fh,
+  wt_off_t offset, size_t len, const void *buf)
+{
+    __wt_verbose_debug3(session, WT_VERB_LIVE_RESTORE, "WRITE %s: %" PRId64 ", %" WT_SIZET_FMT,
+      lr_fh->iface.name, offset, len);
+    return (
+      lr_fh->destination->fh_write(lr_fh->destination, (WT_SESSION *)session, offset, len, buf));
+}
+
 /*
  * __live_restore_fh_write_int --
  *     Write to a file. Callers of this function must hold the extent list lock.
```

Comment:
Unrelated to this PR, but while we're here
```suggestion
 *     Write to a file. Callers of this function must hold the file lock.
```